cmd/kubeadm/app/phases/etcd/local.go

```diff
@@ -247,6 +248,14 @@ func GetEtcdPodSpec(cfg *kubeadmapi.ClusterConfiguration, endpoint *kubeadmapi.A
 				staticpodutil.NewVolumeMount(etcdVolumeName, cfg.Etcd.Local.DataDir, false),
 				staticpodutil.NewVolumeMount(certsVolumeName, cfg.CertificatesDir+"/etcd", false),
 			},
+			Resources: v1.ResourceRequirements{
```

Comment:
@neolit123 looks like we don't need to use `staticpodutil.ComponentResources` here, WDYT?

Reply:
your are correct.
we don't need the utility function and the method i outlined in the issue.